app/server/views/chapter.html.erb

```diff
@@ -0,0 +1,23 @@
+<header id="c-global-header" class="c-book-header" role="banner"><p><a href="/">Home</a></p></header>
+<div class="l-admin">
+    <%= erb :'_table_of_contents.html', locals: { book: book } %>
```

Comment:
Not that it's a huge issue, but this template file is using 4 space indentations instead of 2.

Reply:
Oh, my bad–I'll get that fixed. Thanks!